python/cudf/cudf/tests/test_doctests.py

```diff
@@ -10,6 +10,8 @@
 
 import cudf
 
+pytestmark = pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
We could do that, but I didn't see much value in it. Currently IIRC the list would be cudf, pandas, and numpy (pandas uses a deprecated numpy API somewhere in one example), but in principle it could also come from any other deprecated API that pandas calls and doesn't intercept the warning for, for instance. Given how much of our examples are copied straight from pandas, and since in examples we don't have the flexibility to insert things like `warnings.simplefilter` where needed, I don't think it's worthwhile to overthink how we suppress the warnings. Dealing with those when they break seems good enough to me.